File: Arcade/Intro/javascript/makeArrayConsecutive2.js
function makeArrayConsecutive2(statues) {
    let o = 0;
    statues.sort((a, b) => b - a);    
    for(let i = 0; i < statues.length - 1; i++) {
        o += statues[i] - statues[i + 1] - 1;
    }
    return o;
}

File: Arcade/Intro/Java/MakeArrayConsecutive2.java
import java.util.Arrays;
import java.util.stream.IntStream;

final class MakeArrayConsecutive2 {

	int makeArrayConsecutive2(int[] statues) {
	    Arrays.sort(statues);
	    return IntStream.rangeClosed(statues[0], statues[statues.length - 1])
	        .toArray().length - statues.length;
	}
}